eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -427,6 +427,12 @@
           "code": "java\\.annotation\\.removed",
           "old": ".*? com\\.azure\\.search\\.documents(\\.indexes)?\\.models.*",
           "justification": "Removing Jackson annotations from Search in transition to stream-style."
+        },
+        {
+          "regex": true,
+          "code": "java\\.class\\.externalClassExposedInAPI",
+          "new": "(interface|enum) com\\.azure\\.spring\\.cloud\\.appconfiguration\\.config.*",
+          "justification": "Public class exposed in API"
```

Comment:
Which one is exposed?

Reply:
`AppConfigurationRefresh` and `AppCOnfigurationStoreHealth`. Both are used by `spring-cloud-azure-actuator-autoconfigure`.

Reply:
This happens on any library that uses them in my experience. I have had to make updates for the SDK packages too.

Reply:
I have updated to the new way to do this. The package name change broke it.